### PR TITLE
Fix UpdateAccount to accept optional arguments for partial updates

### PR DESCRIPTION
**Pull request by aalhendi**
This fixes issue    #69 .

The inputs are first made optional by setting ``required = False``. This causes an overwrite of the previous state of the account. In order to update only the inputs being passed from the mutation, the current values are appended in the mixin as kwargs before saving

### CODE REVIEW

**Comment by haris1998-web** [comment]
@PedroBern @ulgens @pors Kindly take a look at this fix as it is much needed.

**Comment by ulgens** [comment]
I feel like the solution is a bit hacky even if it works, and I didn't want to merge without any other maintainer's approval. If there is no objection in the next couple of days, I can merge it.

**Review by ulgens — APPROVED** [review]

**Pull request merged**
